searx/plugins/hostnames.py

```diff
@@ -1,7 +1,21 @@
 # SPDX-License-Identifier: AGPL-3.0-or-later
 # pylint: disable=too-many-branches
-"""In addition to rewriting/replace reslut URLs, the *hoostnames* plugin offers
-other features.
+""" .. note::
+
+    The **Hostname replace** plugin has been deprecated and replace by
+    **Hostnames plugin**.
+
+The **Hostnames plugin** can be enabled by adding it to the
+``enabled_plugins`` **list** in the ``setting.yml`` like so.
+
+  .. code:: yaml
+
+     enabled_plugins:
+       - 'Hostnames plugin'
+       ...
+
+In addition to rewriting/replacing result URLs, the **Hostnames plugin** offers
+additional features.
```

Comment:
"In addition" .. to what? .. implies a reference to the old plugin... but in my opinion we don't need that / the sentence can be deleted.

Reply:
That is what the original text said. It is also unclear to me what is intended here. I will drop it.